lib/commission-math.js

```diff
@@ -3,10 +3,10 @@ const configManager = require('./new-config-manager')
 const { utils: coinUtils } = require('lamassu-coins')
 
 function truncateCrypto (cryptoAtoms, cryptoCode) {
-  const DECIMAL_PLACES = 3
+  const DECIMAL_PLACES = 6
```

Comment:
Doesn't this change affect line 13? I'm not sure, but I'm not understanding how the changes on line 9 compensate this one

Reply:
I got seriously confused when I saw that we use display units to compute crypto values hehe. About your comment: the difference between unitScale and DECIMAL_PLACES is the same as before, so this doesn't change anything in practice, but we have to do it because we now have multiple display scales and they should be used for display purposes only.